src/app/search/helpers.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ *    Copyright 2024 OICR, UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+export function parseTerms(searchText: string): string[] {
+  // Split the terms, conserving double-quoted strings.
```

Comment:
Question, does https://github.com/dockstore/dockstore-ui2/blob/2.12.4/package-lock.json#L3242 mean that https://www.npmjs.com/package/split-string is already available to our code?
What are the trade-offs between using it and this?

Reply:
Switched to split-string module, version `3.1.0` as it is compatible with other components and the most-used released.